h2o-bindings/bin/custom/python/gen_anovaglm.py

```diff
@@ -55,3 +56,103 @@ def result(self):
+>>> x = ['AGE', 'VOL', 'DCAPS']
+>>> y = 'CAPSULE'
+>>> anova_model = H2OANOVAGLMEstimator(family='binomial',
+...                                    lambda_=0,
+...                                    missing_values_handling="skip",
+...                                    highest_interaction_term=2)
+>>> anova_model.train(x=x, y=y, training_frame=train)
+>>> anova_model.summary()
+""",
+    link="""
+>>> import h2o
+>>> h2o.init()
+>>> from h2o.estimators import H2OANOVAGLMEstimator 
+>>> train = h2o.import_file("http://s3.amazonaws.com/h2o-public-test-data/smalldata/prostate/prostate_complete.csv.zip")
+>>> x = ['AGE', 'VOL', 'DCAPS']
+>>> y = 'CAPSULE'
+>>> anova_model = H2OANOVAGLMEstimator(family='binomial',
+...                                    lambda_=0,
+...                                    missing_values_handling="skip",
+...                                    link="family_default")
+>>> anova_model.train(x=x, y=y, training_frame=train)
+>>> anova_model.summary()
+""",
+    nparallelism="""
+>>> import h2o
+>>> h2o.init()
+>>> from h2o.estimators import H2OANOVAGLMEstimator 
+>>> train = h2o.import_file("http://s3.amazonaws.com/h2o-public-test-data/smalldata/prostate/prostate_complete.csv.zip")
+>>> x = ['AGE', 'VOL', 'DCAPS']
+>>> y = 'CAPSULE'
+>>> anova_model = H2OANOVAGLMEstimator(family='binomial',
+...                                    lambda_=0,
+...                                    missing_values_handling="skip",
+...                                    nparallelism=4)
+>>> anova_model.train(x=x, y=y, training_frame=train)
+>>> anova_model.summary()
+""",
+    prior="""
+>>> import h2o
+>>> h2o.init()
+>>> from h2o.estimators import H2OANOVAGLMEstimator 
+>>> train = h2o.import_file("http://s3.amazonaws.com/h2o-public-test-data/smalldata/prostate/prostate_complete.csv.zip")
+>>> x = ['AGE', 'VOL', 'DCAPS']
+>>> y = 'CAPSULE'
+>>> anova_model = H2OANOVAGLMEstimator(family='binomial',
+...                                    lambda_=0,
+...                                    missing_values_handling="skip",
+...                                    prior=0.0)
+>>> anova_model.train(x=x, y=y, training_frame=train)
+>>> anova_model.summary()  
+""",
+    save_transformed_framekeys="""
+>>> import h2o
+>>> h2o.init()
+>>> from h2o.estimators import H2OANOVAGLMEstimator 
+>>> train = h2o.import_file("http://s3.amazonaws.com/h2o-public-test-data/smalldata/prostate/prostate_complete.csv.zip")
+>>> x = ['AGE', 'VOL', 'DCAPS']
+>>> y = 'CAPSULE'
+>>> anova_model = H2OANOVAGLMEstimator(family='binomial',
+...                                    lambda_=0,
+...                                    missing_values_handling="skip",
+...                                    save_transformed_framekeys=False)  
+>>> anova_model.train(x=x, y=y, training_frame=train)
+>>> anova_model.summary()
+""",
+    tweedie_link_power="""
+>>> import h2o
```

Comment:
remove lines 130 to 157, tweedie_link_power and tweedie_variance_power are not ANOVAGLM specific parameters.  They should be made into examples in GLM and not here.